homeassistant/components/mobile_app/entity.py

```diff
@@ -58,7 +60,7 @@ def should_poll(self) -> bool:
     @property
     def name(self):
         """Return the name of the mobile app sensor."""
-        return self._config[ATTR_SENSOR_NAME]
+        return self._name
```

Comment:
For the name, I think the format Sensor Name (Device Name) works better too. That's a suggestion, not a directive, so i'm open to leaving it as is.

Reply:
I think that we should Device Name Sensor Name, since we've been adding logic to strip off start of names if they match the container name. (ie Living Room area card will rename Living Room Lights -> Lights)

Reply:
Also we have a unique ID, let the user do their thing :)

Reply:
Yup I agree we not only have unique ID but we also have the entity registered to the device so its not too difficult to find from device page.  I think users are just confused by the generic name it gets when the sensor gets registered.  

So the format I chose is good?  Not sure if we have a official format to follow :)

Reply:
What Paulus says goes, i'm fine with it.